package.json

```diff
@@ -27,10 +27,9 @@
   },
   "dependencies": {
     "@enact/dev-utils": "^6.1.0",
-    "@storybook/addon-actions": "^6.5.16",
-    "@storybook/addon-controls": "^6.5.16",
-    "@storybook/addon-docs": "^6.5.16",
-    "@storybook/addon-toolbars": "^6.5.16",
+    "@storybook/addon-actions": "^8.1.4",
```

Comment:
Storybook just released 8.1.6 version :( Would you please update? I'm about to merge this PR.